test/unified_format.py

```diff
@@ -611,7 +611,10 @@ def match_result(self, expectation, actual, in_recursive_call=False):
             return self._match_document(expectation, actual, is_root=not in_recursive_call)
 
         if isinstance(expectation, abc.MutableSequence):
-            self.test.assertIsInstance(actual, abc.MutableSequence)
+
+            self.test.assertIsInstance(
+                actual, (abc.MutableSequence, pymongo.command_cursor.CommandCursor)
```

Comment:
This is necessary because `list_indexes` returns a CommandCursor, not a MutableSequence like expected. We do not seem to test the type of the output of list_indexes elsewhere in the spec tests, and in the pymongo specific tests we always convert to list immediately.